src/Package/Impl/Repl/Commands/ReplShortcutSetting.cs

```diff
@@ -2,91 +2,39 @@
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
 using System;
-using System.Diagnostics;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
 using EnvDTE;
-using Microsoft.R.Editor.Settings;
 using Microsoft.VisualStudio.R.Package.Shell;
 
 namespace Microsoft.VisualStudio.R.Package.Repl.Commands {
     internal static class ReplShortcutSetting {
-        private const string CommandName = "EditorContextMenus.CodeWindow.ExecuteLineInInteractive";
-
-        private static bool _currentSetting;
+        private static Dictionary<string, string> _editorShortcutRemaps = new Dictionary<string, string>() {
+            { "Edit.LineOpenAbove", "Text Editor::ctrl+k,ctrl+o" }, // Make Ctrl+Enter work in R Editor
```

Comment:
As for RS defaults, it appears that typical users want familiar ones. Also, quite a few didn't find DS profile.

Generally we only assign them to new commands except Ctrl+Enter and Ctrl+L so it doesn't really matter which shortcuts to use. So we leave making custom setting to advanced folks who can create their own profile.